main/templates/coordinates.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'main.html' %}
+
+{% block coordinates%}
+
+<div class="row d-flex justify-content-center">
+    <h1 class="p-3 text-center">
+        <a href="/" style="color: inherit; text-decoration: none;">GeoLocator</a>
+    </h1>
+</div>
+
+<div class="col-12 d-flex justify-content-start">
+    <button type="button" class="btn btn-sm btn-primary" onclick="redirectToName()">Name</button>
+</div>
+
+<script>
+    function redirectToName() {
```

Comment:
Perhaps you can use `href` tag instead having this function? Something like `href={% url 'homepage' %}` (not sure about syntax).